core/txpool/legacypool/legacypool.go

```diff
@@ -1768,13 +1769,13 @@ func (pool *LegacyPool) demoteUnexecutables() {
 }
 
 func (pool *LegacyPool) startPeriodicFeeUpdate() {
-	if pool.chainconfig.ApricotPhase3BlockTimestamp == nil {
+	if params.GetExtra(pool.chainconfig).ApricotPhase3BlockTimestamp == nil {
```

Comment:
nit given params.GetExtra(pool.chainconfig) is used twice, let's assign to a `config` variable

Reply:
Bundled in #862